packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/__snapshots__/SelectedPeopleList.test.tsx.snap

```diff
@@ -218,7 +218,7 @@ Array [
               & .contextualHost {
                 display: none;
               }
-              &:hover $primaryText {
+              & .ms-Persona-primaryText {
```

Comment:
Is this change expected? Previously these styles were only applied in `:hover` pseudo selector state.

There's a slight difference in output (PR deploy element pane is left-most in screenshot below):

![image](https://user-images.githubusercontent.com/706967/60531718-6db0ef00-9cb0-11e9-80bf-166d6fa6723e.png)

Reply:
Note that the difference isn't noticeable via my 👀 when testing.. seems like someone provided an override when the selector you've now fixed wasn't working. Seems isolated to this component tho, I didn't spot others.